.storybook/pages/ProjectOverview/ProjectOverview.stories.tsx

```diff
@@ -10,5 +10,10 @@ export default {
 
 const Template: Story<Props> = (args) => <ProjectOverview {...args} />;
 
-export const StudentView = Template.bind({});
-StudentView.args = {};
+export const Default = Template.bind({});
+Default.args = {};
+
+export const Sequence = Template.bind({});
+Sequence.args = {
+  listVariant: 'sequence'
```

Comment:
Change this to `Ordered` instead of `Sequence` when you make that prop change. Thanks!